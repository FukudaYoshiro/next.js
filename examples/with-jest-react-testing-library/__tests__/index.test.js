/* eslint-env jest */

import React from 'react'
import { render } from 'react-testing-library'

import App from '../pages/index.js'

describe('With React Testing Library', () => {
  it('Shows "Hello world!"', () => {
    const { getByText } = render(<App />)

    expect(getByText('Hello World!')).not.toBeNull()
  })
})

describe('With React Testing Library Snapshot', () => {
  it('Should match Snapshot', () => {
    const { asFragment } = render(<App />)

    expect(asFragment()).toMatchSnapshot()
  })
})
