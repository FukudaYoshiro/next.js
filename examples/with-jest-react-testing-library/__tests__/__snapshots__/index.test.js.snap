// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`With React Testing Library Snapshot Should match Snapshot 1`] = `
<DocumentFragment>
  <div>
    <p>
      Hello World!
    </p>
  </div>
</DocumentFragment>
`;
